ansible/roles/backstage/tasks/main.yml

```diff
@@ -4,10 +4,13 @@
     name: backstage
   become: yes
 
+- name: install boto3
+  command: "sudo pip install boto3"
```

Comment:
was this missing? It should already be installed [here](https://github.com/1NTERRUPT/terraform/blob/master/modules/control/main.tf#L295)